pkg/storage/objects/store.go

```diff
@@ -24,10 +24,13 @@ var mon = monkit.Package()
 // Meta is the full object metadata
 type Meta struct {
 	pb.SerializableMeta
-	Modified   time.Time
-	Expiration time.Time
-	Size       int64
-	Checksum   string
+	Modified         time.Time
+	Expiration       time.Time
+	Size             int64
+	Checksum         string
+	SegmentsSize     int64
+	RedundancyScheme storj.RedundancyScheme
```

Comment:
this is so weird but we do have a bit of a mismatch where the redundancy scheme is technically per segment and not per object, so that's weird. this is fine though

Reply:
Yeah, I was wondering about that. Do we actually want to support different redundancy between segments of an object? Unless we have tests for it already, I suspect it won't work right yet.

What if we renamed this field to LastRedundancyScheme to try and make clear that it only really applies to one of the segments?

Reply:
no i don't think we do want to support it but it's kind of an unfortunate property of our data model. i think it's fine as is but maybe it's worth a comment?

Reply:
Ok. I'll get that in as part of #1568.